corehq/apps/reports/static/reports/js/filters/case_list_explorer.js

```diff
@@ -68,6 +68,7 @@ hqDefine("reports/js/filters/case_list_explorer", ['jQuery', 'underscore', 'knoc
 
         self.removeProperty = function (property) {
             self.properties.remove(property);
+            $('#fieldset_explorer_columns').trigger('change');
```

Comment:
Similarly to the above comment about `val`, what's this going to be used for, and should it use a binding instead of triggering events itself? The main thing catching my eye is that our knockout code doesn't typically have jquery selectors or act directly on DOM elements.

Reply:
so the "Apply" button on the reports page relies on a `change` event to happen on this div to become "active". The only way I can make that happen is by directly triggering this here. Maybe there is another way, but I couldn't think of one, since that button is directly listening for this event, and the element that is being removed no longer exists.